src/Glpi/Inventory/Conf.php

```diff
@@ -384,26 +384,63 @@ public function showConfigForm()
             echo '</div>';
             echo "</td>";
             echo "</tr>";
-
             echo "<tr class='tab_bg_1'>";
             echo "<td>";
             echo "<label for='auth'>" . __s('Authorization header') . "</label>";
             echo "</td>";
             echo "<td>";
             Dropdown::showFromArray('auth_required', [
                 'none' => __('None'),
-                'client_credentials' => __('OAuth - Client credentials')
+                'client_credentials' => __('OAuth - Client credentials'),
+                'basic_authentification' => __('Basic Authentification'),
             ], [
                 'value' => $config['auth_required'] ?? 'none'
             ]);
             echo "</td></tr>";
+            echo "<tr class='tab_bg_1' id='basic_auth_login_row'>";
+            echo "<td>";
+            echo "<label for='basic_auth_login'>" . __s('Login') . "</label>";
+            echo "</td>";
+            echo "<td>";
+            echo Html::input("basic_auth_login", [
+                "value" => $config["basic_auth_login"]
+            ]);
+            echo "</td>";
+            echo "</tr>";
+            echo "<tr class='tab_bg_1' id='basic_auth_password_row'>";
+            echo "<td>";
+            echo "<label for='basic_auth_password'>" . __s('Password') . "</label>";
```

Comment:
```suggestion
            echo "<label for='basic_auth_password'>" . __('Password') . "</label>";
```
`__s('string')` escape HTML entities (no need here)